core/src/test/scala/unit/kafka/server/ReplicationQuotasTest.scala

```diff
@@ -235,8 +220,25 @@ class ReplicationQuotasTest extends ZooKeeperTestHarness {
   }
 
   private def property(key: String, value: String) = {
-    new Properties() { put(key, value) }
+    new Properties() {
+      put(key, value)
+    }
```

Comment:
Nitpick: subclassing `Properties` like this is a bit odd. It's slightly more concise, but doesn't seem worth it.